pipeline/api-service/overlays/external-mysql/params.env

```diff
@@ -0,0 +1,3 @@
+mysqlHost=
+mysqlUser=
```

Comment:
Temporarily put username  and password to configmap. These will be used to replace placeholder in pipeline configmap. The challenge is the value from secret is encoded string and I can not find way to decode it easily.
secretGenerator can not use `StringData` and upstream issue is tracked here.  https://github.com/kubernetes-sigs/kustomize/issues/1444